src/app/constants/CheckMediaTypes.js

```diff
@@ -0,0 +1,16 @@
+const CheckMediaTypes = {
+  CLAIM: 'Claim',
+  LINK: 'Link',
+  UPLOADEDIMAGE: 'UploadedImage',
+  UPLOADEDVIDEO: 'UploadedVideo',
+  UPLOADEDAUDIO: 'UploadedAudio',
```

Comment:
These also would be better declared in CONSTANT_CASE: `UPLOADED_IMAGE`, `UPLOADED_VIDEO` and `UPLOADED_AUDIO`